classes/article/ArticleGalleyDAO.inc.php

```diff
@@ -493,15 +493,15 @@ function getByPubIdType($pubIdType, $contextId = null, $title = null, $author =
 			FROM	submission_galleys g
 				' . ($contextId != null?' JOIN submissions s ON (s.submission_id = g.submission_id)':'') . '
 				LEFT JOIN published_submissions ps ON (ps.submission_id = g.submission_id)
-				LEFT JOIN issues i ON (ps.issue_id = i.issue_id AND i.date_published IS NOT NULL)
+				LEFT JOIN issues i ON (ps.issue_id = i.issue_id)
```

Comment:
that is ok, I believe -- `getCompletionConditions` does not have any parameters...